larndsim/pixels_from_track_ep.py

```diff
@@ -124,7 +125,7 @@ def get_neighboring_pixels(self, active_pixels, radius, max_pixels):
             n_indices = ep.tile(neighbor_indices, [track.shape[0], 1]) + \
                         ep.tile(track, [1, neighbor_indices.shape[0]]).reshape([-1, 2])
             n_indices = n_indices[:, 0] * self.n_pixels[1] + n_indices[:, 1]
-            n_indices_idx =np.unique(n_indices.raw.numpy(), return_index=True)[1]
+            n_indices_idx =np.unique(n_indices.raw.cpu().numpy(), return_index=True)[1]
```

Comment:
Let's think of an alternative.

Reply:
This is mostly in here for comparison with the non-differentiable sim (preserves some of the ordering) -- I think we can probably do this after the fact and switch back to your ep.unique function, but I'll double check